.changeset/chilly-jokes-decide.md

```diff
@@ -0,0 +1,6 @@
+---
+"@biomejs/backend-jsonrpc": minor
+"@biomejs/biome": minor
+---
+
+Ensure interface members are sorted, for consistency and readability.
```

Comment:
Changesets will end up in the changelog, and end users will read this line and they won't understand what this line is about. Since we're adding a new rule, try to add the name of the rule, a brief explanation of what it does with a code snippet.

And please read the contribution guide,  the PR template has a link exactly to the chapter that talks about what I just said.

See past releases to understand what to write.